lib/msf/core/module_manager/loading.rb

```diff
@@ -116,7 +116,9 @@ def load_modules(path, options={})
 
     loaders.each do |loader|
       if loader.loadable?(path)
-        count_by_type = loader.load_modules(path, options)
+        count_by_type.merge!(loader.load_modules(path, options)) do |key, old, new|
+          old + new
```

Comment:
@acammack-r7 no prob just wanted to make sure this does not break anything inside metasploit